tests/core/snap-set-core-config/task.yaml

```diff
@@ -52,6 +52,12 @@ execute: |
         exit 1
     fi
 
+    echo "Check that ctrl-alt-del-action=none works"
+    snap set core system.ctrl-alt-del-action=none
+    systemctl show --property=LoadState ctrl-alt-del.target | MATCH "LoadState=masked"
+    echo "Check that ctrl-alt-del-action=reboot works"
+    snap set core system.ctrl-alt-del-action=reboot
+    systemctl show --property=UnitFileState ctrl-alt-del.target | MATCH "UnitFileState=disabled"
```

Comment:
for completeness shouldn't this also check --property=LoadState ?

Reply:
Good point, let me add this

Reply:
it should be loaded here/at this point